include/zed_cpu/zed_cpu.hpp

```diff
@@ -0,0 +1,37 @@
+#include <memory>
+#include <vector>
+
+#include <image_transport/image_transport.hpp>
+#include <rclcpp/rclcpp.hpp>
+
+#include <sensor_msgs/msg/imu.hpp>
+
+#include <zed_lib/sensorcapture.hpp>
+#include <zed_lib/videocapture.hpp>
+
+namespace zed_cpu
+{
+class ZedCameraNode
```

Comment:
```suggestion
namespace zed_cpu
{

class ZedCameraNode : public rclcpp::Node
```
For ROS 2 this approach is more common